### DISCUSSION TITLE
Fixed VPN IP Address

### DISCUSSION THREAD

**Discussion by benbelo** [yuezk/GlobalProtect-openconnect#332] [Q&A]
Hello,

I am trying to force gpclient to use a fixed VPN IPv4 address on tun0.
I need it to match my security rules (I don't use user related rules).

I tried to modify the variables inside /etc/vpnc/vpnc-script (CISCO_SPLIT_INC_ADDR).
Maybe I can create a bash script that loops gpclient connect until it gets the right IP address ? (not a good idea I know)

Have you ever had this problem ?

Thank you for your help

**Comment by yuezk** [comment]
I haven’t deal with this problem. But I can try to figure out how the VPN address is determined, to see if it could be specified.

**Reply by benbelo** [reply]
Hello, did you find something about the fixed ip address ? Thank you
